openedx/adg/lms/webinars/tests/constants.py

```diff
@@ -12,3 +12,5 @@
 CO_HOST_2 = 'ch2@eg.com'
 
 PRESENTER = 'presenter@eg.com'
+
+FAKE_MANDRILL_MSG_ID = 'hhdhvdhvdhdvd'
```

Comment:
Lets assign any meaningful value unless it has to be exactly what you have assigned. For example `FAKE_MANDRILL_MSG_ID = 'fake_message_id'`